subsys/bluetooth/controller/CMakeLists.txt

```diff
@@ -84,6 +84,14 @@ if(CONFIG_BT_LL_SW_SPLIT)
       )
     endif()
   endif()
+  if(CONFIG_BT_CTLR_PERIPHERAL_ISO OR
+     CONFIG_BT_CTLR_CENTRAL_ISO OR
+     CONFIG_BT_CTLR_LE_FEATURE)
```

Comment:
I feel it's unclear why `CONFIG_BT_CTLR_PERIPHERAL_ISO` or `CONFIG_BT_CTLR_CENTRAL_ISO` would compile this file. How about having the `CONFIG_*_ISO` Kconfigs `select` `CONFIG_BT_CTLR_LE_FEATURE` instead?

Reply:
Good suggestion, as long as LE_FEATURE can still be enabled without ISO being present

Reply:
The command seems to be quite generic, so I don't see why we wouldn't be able to (or want) to have it as a general improvement :)

Reply:
Implemented the suggested change for split ctrl

Reply:
Two questions:
1) Why only for the split controller? 
2) Is the LE Set Host Feature command actually really something that is required for ISO (since we added `select`)?

Reply:
1. Could be added in other cases I guess, but for now we are working on the split ctrl 
2. The spec lists: C49: Mandatory if LE Feature (Connected Isochronous Stream - Master) or LE Feature
(Connected Isochronous Stream - Slave) is supported, otherwise optional

Reply:
Given that `CONFIG_BT_CTLR_PERIPHERAL_ISO`/`CONFIG_BT_CTLR_CENTRAL_ISO` select `CONFIG_BT_CTLR_LE_FEATURE` in the Kconfig, I think checking them is not needed here.

Reply:
@wopu-ot I guess that depends on other Kconfigs: For a non-split configuration, I guess this still makes sense.

Reply:
@Thalley I am not sure I understand what you mean. Can you elaborate?

Reply:
@wopu-ot `CONFIG_BT_CTLR_PERIPHERAL_ISO/CONFIG_BT_CTLR_CENTRAL_ISO` only `select`ed `CONFIG_BT_CTLR_LE_FEATURE` if `BT_LL_SW_SPLIT` is enabled in this PR.

Reply:
On the one hand, this is in a block that is guarded with `if(CONFIG_BT_LL_SW_SPLIT)`. On the other hand, `BT_LL_SW_SPLIT` is the only possible choice for the Zephyr controller and the file would not be applicable for other controllers.